src/main/resources/reference/human_g1k_v37.json

```diff
@@ -0,0 +1,720 @@
+{
+    "contigs": [
+        {
+            "name": "1",
+            "length": 249250621,
+            "isX": false,
```

Comment:
ah, I see. You wanted it to match the case class definition? 
I still think it should be simplified. It should be as easy as possible for someone else to come along and write a reference json